src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -942,14 +942,13 @@ protected DynamoModel(IStartConfiguration config)
             DynamoModel.OnRequestUpdateLoadBarStatus(new SplashScreenLoadEventArgs(Resources.SplashScreenLoadNodeLibrary, 50));
             InitializeNodeLibrary();
 
-            if (!IsServiceMode && extensions.Any())
+            if (extensions.Any())
```

Comment:
this is to address dynamo player case?

Reply:
yeah I am uncertain if this is needed or not because @saintentropy mentioned that Player will be loaded as an extension, but it seems we can still skip loading package manager?

Reply:
@reddyashish Adding this back to skip loading package manager as we planned, when needed, we can add logic for loading player specifically